### PR TITLE
Added Github Folder and updated links related to it.

### LINKED ISSUE #1207: Merge git and GitHub folder contents => git folder
Please create a new top-level folder called `git` and fill it with the contents below. For any file you move, please search for all existing internal links to the file and update to its new location.

- Move all contents of [/git-version-control](https://github.com/Techtonica/curriculum/tree/master/git-version-control) into this new folder
- Move github-code-storage/github-storage.md
- Move github-collaboration/github-collaboration.md

### PR DESCRIPTION
**Pull request by Yashank18**
Fixes #1207 
@vegetabill Please Review

### CODE REVIEW

**Comment by alodahl** [comment]
Nitpick:  The contributor did a good job, but I feel weird about renaming all of these to `github` when much of the content is about git, and could technically be used without github (like locally, or bitbucket even though we wont be using that).  It's not a huge deal, but I would argue the difference between git and GitHub should be as clear as possible to new apprentices.

Sorry I didn't notice earlier.

**Comment by Yashank18** [comment]
@alodahl So is this PR will be considered or should I close this?

**Comment by vegetabill** [comment]
@alodahl I agree with that goal but I really doubt the name of the folder would create that confusion, precisely because it currently doesn't correct it. If you have suggested revisions for the content itself to make that more clear, please open an issue.

**Comment by vegetabill** [comment]
@Yashank18 l've updated the issue to say that the new folder should be called `git` instead of GitHub which will hopefully reduce any confusion.

While you're revising this, please convert any absolute links you come across to relative links.

e.g.
../git/something.md instead of https://github.com/blob/master...

**Comment by Yashank18** [comment]
Okay @vegetabill

**Comment by Yashank18** [comment]
@vegetabill  Please review

File: projects/recipe-page/phase-2-css-prompt.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ Phase 4 of the project involves incorporating jQuery. You'll add at least one in
 ## Setup Instructions
 Use the Terminal for navigating around the file system and creating new folders and files. Refer to the lesson on [The Command Line Interface](https://github.com/Techtonica/curriculum/blob/master/command-line/command-line-interface.md) if you need guidance on using the Terminal.
```
Comment (Kundan28) [inline projects/recipe-page/phase-2-css-prompt.md]: Make the `The Command Line Interface` link relative.

File: projects/recipe-page/phase-3-bootstrap-prompt.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ Phase 4 of the project involves incorporating jQuery. You'll add at least one in
 ## Setup Instructions
 Use the Terminal for navigating around the file system and creating new folders and files. Refer to the lesson on [The Command Line Interface](https://github.com/Techtonica/curriculum/blob/master/command-line/command-line-interface.md) if you need guidance on using the Terminal.
```
Comment (Kundan28) [inline projects/recipe-page/phase-3-bootstrap-prompt.md]: Make this link relative too.

Comment (Kundan28) [reply projects/recipe-page/phase-3-bootstrap-prompt.md]: Make the link in line 45 relative too.

File: projects/recipe-page/phase-2-css-prompt.md (hunk 1 above)
Comment (Yashank18) [reply projects/recipe-page/phase-2-css-prompt.md]: Am I supposed to edit this link? As I was asked to changed git related files only? @vegetabill

File: git/github-storage.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ About 1.5 hour
 
 ### Prerequisites
 - [Command Line Interface lesson](/command-line/command-line-interface.md)
-- [Git & Version Control lesson](/git-version-control/git-version-control.md)
+- [Git & Version Control lesson](/github/git-version-control.md)
```
Comment (vegetabill) [inline git/github-storage.md]: broken link you can change this to just ./git-version-control.md

File: git/gitignore.md [hunk 4]
```diff
@@ -12,7 +12,7 @@ About 30-45 minutes
 
 Here are links to lessons that should be completed before this lesson:
 
-- [Git version control](/git-version-control/git-version-control.md)
+- [Git version control](/github/git-version-control.md)
```
Comment (vegetabill) [inline git/gitignore.md]: fix

File: projects/recipe-page/phase-1-html-prompt.md [hunk 5]
```diff
@@ -42,7 +42,7 @@ Phase 4 of the project involves incorporating jQuery. You'll add at least one in
 ## Setup Instructions
 Use the Terminal for navigating around the file system and creating new folders and files. Refer to the lesson on [The Command Line Interface](https://github.com/Techtonica/curriculum/blob/master/command-line/command-line-interface.md) if you need guidance on using the Terminal.
 
-When you get to the steps below that ask you to initialize a git repo and track files using git, refer to the lesson on [Git and Version Control](https://github.com/Techtonica/curriculum/blob/master/git-version-control/git-version-control.md) if you need guidance.
+When you get to the steps below that ask you to initialize a git repo and track files using git, refer to the lesson on [Git and Version Control](../git/git-version-control.md) if you need guidance.
```
Comment (vegetabill) [inline projects/recipe-page/phase-1-html-prompt.md]: Thanks for relative-izing this!

**Comment by Yashank18** [comment]
@vegetabill

**Comment by Yashank18** [comment]
@vegetabill please review so that I can take up other issue.

File: projects/recipe-page/phase-3-bootstrap-prompt.md (hunk 2 above)
Comment (alodahl) [reply projects/recipe-page/phase-3-bootstrap-prompt.md]: I don't think this is in-scope, Kundan - this issue is just about git directory links.

**Comment by Yashank18** [comment]
@alodahl  thank you for merging. Can you please add Gssoc label and other tag so that my score can be updated

**Comment by alodahl** [comment]
https://github.com/Techtonica/curriculum/issues/1207#issuecomment-599176751

**Comment by Kundan28** [comment]
> [#1207 (comment)](https://github.com/Techtonica/curriculum/issues/1207#issuecomment-599176751)
> 
> I will add them to the final PR we discussed once the issue is complete.

In the scoresheet we have to keep this PR number or the one that @Yashank18 would make later ? @alodahl

**Comment by alodahl** [comment]
@Kundan28 sorry I missed this.  We should be recording the PR that actually gets merged.  If it takes two for some reason, they should be combined (like this, maybe?  "(#000 & #001)") so we don't accidentally give them double points.

**Comment by Kundan28** [comment]
> @Kundan28 sorry I missed this. We should be recording the PR that actually gets merged. If it takes two for some reason, they should be combined (like this, maybe? "(#000 & #000)") so we don't accidentally give them double points.

Okay, I'll keep track of it,

**Comment by Kundan28** [comment]
> @Kundan28 sorry I missed this. We should be recording the PR that actually gets merged. If it takes two for some reason, they should be combined (like this, maybe? "(#000 & #000)") so we don't accidentally give them double points.

But in this case there's only one PR, right ?

**Comment by alodahl** [comment]
You would know better than me 😅

**Comment by Kundan28** [comment]
> You would know better than me 😅

I don't think so 😄 
As you are having more experience than me here.

**Pull request merged**